scripts/multiplatform_build.sh

```
@@ -0,0 +1,362 @@
+        build_cmd="$build_cmd --no-default-features --features $features"
+    fi
+    
+    print_status "Executing: $build_cmd"
+    
+    if eval "$build_cmd"; then
+        print_success "Build completed for $target"
+        
+        # Copy binary to artifacts directory
+        local binary_name="lavalink-rust"
+        local source_path="$BUILD_DIR/$target/release/$binary_name"
+        local dest_name="lavalink-rust-$(echo "$target" | sed 's/unknown-//' | sed 's/-gnu//' | sed 's/-musl/-musl/')"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)